src/wizard/ActionBox.tsx

```diff
@@ -20,20 +20,20 @@ interface Props {
 export class ActionBox extends Component<Props> {
   render = (): React.ReactElement => {
     return (
-      <Page className={this.props.first ? 'ActionBox' : 'ActionBox RightArrow'}>
-        <div className="ups-number">
+      <Card className={this.props.first ? 'ActionBox' : 'ActionBox RightArrow'}>
+        <div className="ups-number" style={{ textAlign: 'center' }}>
```

Comment:
Can you please add the `textAlign: 'center'` to the `ups-number` class and remove it from here?
Passing styles using the `style` attribute should be avoided